### PR TITLE
Fix and test RHEL 8 compatibility of bridge connector

### LINKED ISSUE #53: test/fix target machine on RHEL 8
I think I saw this crash in cockpit-bridge-websocket-connector the other day, but didn't pay much attention to it. Let's move server/Containerfile to CentOS 8 and test with that.

### PR DESCRIPTION
**Pull request by martinpitt**
Fixes #53

### CODE REVIEW

File: server/Containerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Test container image that represents a remote managed server which connects
 # to consoledot. This illustrates the minimum dependencies.
-FROM debian:bookworm
 
-RUN apt-get update && \
-    apt-get install -y cockpit-bridge cockpit-system ca-certificates python3 && \
-    apt-get clean
+ARG tag=stream9
```
Comment (jelly) [inline server/Containerfile:4]: Nice! Cool that we test 8/9 now!

Comment (jelly) [reply server/Containerfile:4]: But this is a bit inconsistent? By default we test on centos8 in the Makefile so wouldn't that make a better default?

File: test/test_basic.py [hunk 2]
```diff
@@ -107,7 +107,7 @@ def wait_status(self, sessionid, expected_status, iterations=10):
         else:
             self.fail(f'session status was not updated to {expected_status}, still at {status}')
 
-    def newSession(self):
+    def newSession(self, tag='latest'):
```
Comment (jelly) [inline test/test_basic.py:110]: Hmm what would latest mean here? centos9? Shouldn't we be explicit.

File: server/Containerfile (hunk 1 above)
Comment (martinpitt) [reply server/Containerfile:4]: We don't, it's centos9 -- that's precisely what this ARG says. But indeed it's confusing to have webconsoleserver:latest be centos9, I'll make the tags consistent.

Comment (martinpitt) [reply server/Containerfile]: Done

File: test/test_basic.py (hunk 2 above)
Comment (martinpitt) [reply test/test_basic.py]: You are right, done.

**Review by jelly — APPROVED** [review]

**Pull request merged**